compiler/ir/ir.tree/src/org/jetbrains/kotlin/ir/descriptors/IrBuiltIns.kt

```diff
@@ -251,4 +251,20 @@ class IrBuiltIns(
     companion object {
         val KOTLIN_INTERNAL_IR_FQN = FqName("kotlin.internal.ir")
     }
+
+    object OperatorNames {
+        const val OPERATOR_NAME_LESS = "less"
+        const val OPERATOR_NAME_LESS_OR_EQUAL = "lessOrEqual"
+        const val OPERATOR_NAME_GREATER = "greater"
```

Comment:
@ting-yuan Please remove also OPERATOR_NAME prefixes (it wasn't clearly mentioned but presented in suggested example in https://github.com/JetBrains/kotlin/pull/2111/files#r264669713)